### PR TITLE
Search in current conversation (fixed selector)

### PR DESCRIPTION
**Pull request by Code-Curious**

### CODE REVIEW

File: src/messenger-shortcuts.js [hunk 1]
```diff
@@ -171,7 +171,7 @@ function sendLike() {
 }
 
 function searchInConversation() {
-  var targetNode = getByAttr('em', 'data-intl-translation', 'Search in Conversation');
+  var targetNode = document.querySelector('._3szn._3szo ._5odt') // return the first one that matches the selector
   click(targetNode);
```
Comment (sarangjo) [inline src/messenger-shortcuts.js]: I would recommend surrounding this call to `click()` with a null-check, since the info pane might not always be open!

Comment (guoguo12) [inline src/messenger-shortcuts.js]: @Code-Curious: Thanks for the fix. I'm not enthusiastic about hardcoding non-semantic DOM selectors (like `._3szn._3szo ._5odt`) into the extension. Is there any other way to do it?

Also, @sarangjo has a good point.

Comment (Code-Curious) [inline src/messenger-shortcuts.js]: @sarangjo you are absolutely right.
@guoguo12 unfortunately I couldn't find a semantic selector, it seems that the rendered DOM for the controls in the info pane is now just a bunch of divs with classes and `role=button`

**Comment by guoguo12** [comment]
Cool, thanks @Code-Curious. I've merged it via a cherry-pick (ee82f89cf3b2c4d4c205c7380ec4870849ee915e).

File: src/messenger-shortcuts.js (hunk 1 above)
Comment (guoguo12) [inline src/messenger-shortcuts.js]: Alright, thanks for trying.

**Pull request closed without merging**